hasher-matcher-actioner/hmalib/common/models/bank.py

```diff
@@ -392,7 +404,11 @@ def __init__(self, table: Table):
         self._table = table
 
     def create_bank(
-        self, bank_name: str, bank_description: str, is_active: bool = True
+        self,
+        bank_name: str,
+        bank_description: str,
+        is_active: bool = True,
```

Comment:
nit/previous PR: should create bank default to `is_active` as `False`

Reply:
The UI does default to False, but for consistency's sake, will change.